sdk/storage/inc/datalake/directory_client.hpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "common/storage_credential.hpp"
+#include "common/storage_url_builder.hpp"
+#include "datalake/path_client.hpp"
+#include "datalake_options.hpp"
+#include "http/pipeline.hpp"
+#include "protocol/datalake_rest_client.hpp"
+
+#include <memory>
+#include <string>
+
+namespace Azure { namespace Storage { namespace Files { namespace DataLake {
+
+  struct DirectoryRenameResponse
+  {
+    std::string Date;
+    Azure::Core::Nullable<std::string> ETag;
+    Azure::Core::Nullable<std::string> LastModified;
+    std::string RequestId;
+    std::string Version;
+    Azure::Core::Nullable<std::string> Continuation;
+  };
+
+  using DirectorySetAccessControlRecursiveResponse = PathSetAccessControlRecursiveResponse;
+  using DirectoryInfo = PathInfo;
+  using DirectoryCreateOptions = PathCreateOptions;
+  using DirectoryDeleteResponse = PathDeleteResponse;
+
+  class DirectoryClient : public PathClient {
+  public:
+    /**
+     * @brief Create from connection string
+     * @param connectionString Azure Storage connection string.
+     * @param fileSystemName The name of a file system.
+     * @param path The path of a resource within the file system.
```

Comment:
Check the parameter name in comments. Ditto for others

Reply:
You resolved this comment but you didn't make the change.

Reply:
Made sure that this is changed this time.